webpack.config.js

```diff
@@ -293,6 +293,7 @@ var config = {
                 {from: 'images/logo_email_dark.png', to: 'images'},
                 {from: 'images/logo_email_gray.png', to: 'images'},
                 {from: 'images/forgot_password_illustration.png', to: 'images'},
+                {from: 'images/forgot_password_illustration_new.png', to: 'images'},
```

Comment:
Instead of adding this new file and leaving the now unused old one still there, can we replace the old picture with the new one and keep the same name?

Reply:
Hey @neallred . I have updated the code.